rules/aws/exfiltration_ec2_vm_export_failure.toml

```diff
@@ -0,0 +1,55 @@
+[metadata]
+creation_date = "2021/04/22"
+maturity = "production"
+updated_date = "2021/04/22"
+
+[rule]
+author = ["Elastic", "Austin Songer"]
+description = """
+An attempt to export an AWS EC2 instance has been detected and a VM Export might indicate an attempt to extract
+information from an instance.
+"""
+false_positives = ["Adding User Keys to their own Accounts"]
```

Comment:
```suggestion
false_positives = [
    """
    VM exports may be done by a system or network administrator. Verify whether the user identity, user agent, and/or
    hostname should be making changes in your environment. VM exports from unfamiliar users or hosts should be
    investigated. If known behavior is causing false positives, it can be exempted from the rule.
    """,
]
```

Reply:
re-lint